src/java/com/puppetlabs/http/client/impl/PersistentSyncHttpClient.java

```diff
@@ -26,10 +31,42 @@ private static void logAndRethrow(String msg, Throwable t) {
         throw new HttpClientException(msg, t);
     }
 
+    public Map<String, Timer> getClientMetrics(){
+        return Metrics.getClientMetrics(metricRegistry);
+    }
+
+    public Map<String, Timer> getClientMetrics(String url, Metrics.MetricType metricType) {
```

Comment:
I think at one point @cprice404 and I discussed this and decided that we leaned towards having these methods take the client (rather than require the metric registry to be passed in) because consumers of this already have to be holding on to a reference to the client. That said, I don't feel strongly about this, and if we do make it so that basic sync requests that don't use a client can have metrics tracked on them then we definitely need to change this. It would make testing easier too.